datalad/distribution/drop.py

```diff
@@ -52,25 +56,39 @@
     dest='check')
 
 
-res_kwargs = dict(action='drop', logger=lgr)
-
-
-def _drop_files(ds, files, check, noannex_iserror=True):
+def _drop_files(ds, files, check, noannex_iserror=True, **kwargs):
+    if 'action' not in kwargs:
```

Comment:
Would be nice to have a little docstring for those kind of helpers in general.
May be not sth one would condier a "full" documentation, but a sentence on what it is supposed to do and 3 words per parameter or sth like that.